tests/system/action/organization/test_initial_import.py

```diff
@@ -0,0 +1,531 @@
+from typing import Any, Dict
+
+from tests.system.action.base import BaseActionTestCase
+
+
+class OrganizationInitialImport(BaseActionTestCase):
+    def create_request_data(self, datapart: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
You can remove the `datapart` if it is not used anywhere

Reply:
Removed.